src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/messenger.html.twig

```diff
@@ -0,0 +1,62 @@
+{% extends '@WebProfiler/Profiler/layout.html.twig' %}
+
+{% import _self as helper %}
+
+{% block toolbar %}
+    {% set color_code = 'normal' %}
+    {% set message_count = 0 %}
+    {% set icon %}
+        {{ include('@WebProfiler/Icon/messenger.svg') }}
+        <span class="sf-toolbar-value">{{ message_count }}</span>
+    {% endset %}
+
+    {{ include('@WebProfiler/Profiler/toolbar_item.html.twig', { link: 'messages', status: color_code }) }}
+{% endblock %}
+
+{% block menu %}
+<span class="label">
+    <span class="icon">{{ include('@WebProfiler/Icon/messenger.svg') }}</span>
+    <strong>Messages</strong>
+
+    {% if collector.messages|length > 0 %}
```

Comment:
IIRC, we removed all counts from the profiler menu in the redesign done in 2.8. Why having it here ?

Reply:
True ... but I've moved this to `{% block toolbar %}` because the Messenger must only be displayed if there are messages, like we do with the rest of panels.

Reply:
Well, you haven't removed the count itself in the profiler menu